app/index.tsx

```diff
@@ -1,15 +1,21 @@
+import MuiThemeProvider from "material-ui/styles/MuiThemeProvider";
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 import { Provider } from "react-redux";
+import * as injectTapEventPlugin from "react-tap-event-plugin";
 import { applyMiddleware, createStore } from "redux";
 import { composeWithDevTools } from "redux-devtools-extension";
 import reduxLogger from "redux-logger";
 import reduxThunk from "redux-thunk";
 import App from "./containers/App";
 import { Commit } from "./containers/Commit";
+import muiTheme from "./muiTheme";
 import reducers from "./reducers";
 
 // @todo add bundle splitting and separate these renders #UAF
+
+injectTapEventPlugin();
```

Comment:
tbh i never used it before. Do we need it still today? https://www.reddit.com/r/reactjs/comments/4pe61k/why_do_we_need_reacttapeventplugin_in_our_projects/ i think material-ui doesnt rely on it anymore

Reply:
fixed